server/views/pages/licence/HDC_AP.njk

```diff
@@ -187,7 +186,6 @@
       </p>
    </div>
 
-   <div class="smallPaddingBottom"></div>
```

Comment:
removed as unused (no corresponding class) and unneeded - originally brought over from HDC.